src/Akeneo/Tool/Bundle/MeasureBundle/Application/CreateMeasurementFamily/CreateMeasurementFamilyHandler.php

```diff
@@ -4,13 +4,15 @@
 
 namespace Akeneo\Tool\Bundle\MeasureBundle\Application\CreateMeasurementFamily;
 
+use Akeneo\Tool\Bundle\MeasureBundle\Event\MeasurementFamilyCreated;
 use Akeneo\Tool\Bundle\MeasureBundle\Model\LabelCollection;
 use Akeneo\Tool\Bundle\MeasureBundle\Model\MeasurementFamily;
 use Akeneo\Tool\Bundle\MeasureBundle\Model\MeasurementFamilyCode;
 use Akeneo\Tool\Bundle\MeasureBundle\Model\Operation;
 use Akeneo\Tool\Bundle\MeasureBundle\Model\Unit;
 use Akeneo\Tool\Bundle\MeasureBundle\Model\UnitCode;
 use Akeneo\Tool\Bundle\MeasureBundle\Persistence\MeasurementFamilyRepositoryInterface;
+use Psr\EventDispatcher\EventDispatcherInterface;
```

Comment:
Is there a reason to not use Symfony's interface? (I know it uses PSR under the hood, just askinnnnnnn)
```suggestion
use Symfony\Component\EventDispatcher\EventDispatcherInterface;
```

Reply:
I think it's equivalent, Psr is more "general" than symfony (i'm good either way)